### PR TITLE
PLAT-451: Access to conveyor cache

### PR DESCRIPTION
**Pull request by cyraxred**
**- What I did**
* Access to conveyor cache
* Change cache to store pulse.Range instead of pulse.Data
* Convenience functions to get prev pulse data

### CODE REVIEW

File: ledger-core/conveyor/conveyor.go [hunk 1]
```diff
@@ -103,6 +103,10 @@ type PulseConveyor struct {
 	stoppingChan <-chan struct{}
 }
 
+func (p *PulseConveyor) GetDataManager() *PulseDataManager {
```
Comment (cyraxred) [inline ledger-core/conveyor/conveyor.go:106]: @ivanshibitov here is the first function you need

File: ledger-core/conveyor/pulse_manager.go [hunk 2]
```diff
@@ -86,6 +87,21 @@ func (p *PulseDataManager) GetPresentPulse() (present pulse.Number, nearestFutur
 	return p._split(v)
 }
 
+func (p *PulseDataManager) GetPrevPulseRange() (pulse.Number, pulse.Range) {
```
Comment (cyraxred) [inline ledger-core/conveyor/pulse_manager.go:90]: @ivanshibitov here is the second function you need

File: ledger-core/conveyor/conveyor.go (hunk 1 above)
Comment (ivanshibitov) [inline ledger-core/conveyor/conveyor.go:106]: It's not clear what is 'DataManager'. May be GetPulseDataManager?

Comment (cyraxred) [reply ledger-core/conveyor/conveyor.go:106]: well, there should be more data in it soon, hence the name

**Pull request merged**